c/httpserver.c

```diff
@@ -5887,6 +5963,20 @@ int mainHttpLoop(HttpServer *server){
                                             httpWorkElementHandler,
                                             httpBackgroundHandler);
 
+  runMonitoringThread();
+  STCModule *heartbeatMonitoringModule = stcRegisterModule(
+        base,
+        STC_MODULE_GENERIC,
+        server,
+        NULL,
+        NULL,
+        monitoringWorkElementHandler,
+        NULL
+    );
+
+  WorkElementPrefix *workElement = makeMonitoringWorkElement();
+  stcEnqueueWork(base, workElement);
```

Comment:
way 2: use stcEnqueue - but keep executing - can sleep between work item - need a way to make stcDrainQueue task asynchronous so i can sleep in task without blocking queue otherwise it thrashes server - if i keep adding work item at end of work item to make a monitoring loop using queue.